CONTRIBUTING.md

```diff
@@ -164,6 +186,51 @@ After you get your container up and running, you can test it by adding test asse
 
 Finally, commit your changes and submit a PR - we'll take a look at it, provide any needed feedback, and then merge it in. We appreciate any and all feedback!
 
+### Adding a Database Definition to an existing Container
+
+VS Code Dev Containers allow multiple containers to run together. This is useful, for example, when adding database support to an existing container, and can be leveraged with `docker compose.`
+
+1. Create a new copy of an existing language definition folder, and hyphenate the database name.
+   1. For example, `java` becomes `java-postgres`.
+2. Create a new `docker-compose.yml` file.
+   1. Add the existing `Dockerfile` to the `docker-compose` as an `app`. This will be the Dev Container where your code gets executed.
+   2. Add the definition for the database to the `docker-compose`, version tagging it or using the `latest`. Most usage scenarios don't require custom images for the database.
+   3. It's common that containers will need to be connected through a network so that the `app` can connect to the `db`. This can normally be done by adding the following to the `app` definition:
+
+    ```yaml
+        network_mode: service:db
+    ```
+
+3. Modify the `devcontainer.json` with any launch options or VS Code specific `settings.json` options that may be needed. 
+   1. For example, when working with a SQL database, it might be necessary to modify the settings with something like the following:
+   ```json
+   		"sqltools.connections": [
+			{
+			  "name": "Container database",
+			  "driver": "PostgreSQL",
+			  "previewLimit": 50,
+			  "server": "localhost",
+			  "port": 5432,
+			  // NOTE: database/username/password should match values in docker.compose.yml
+			  "database": "postgres",
+			  "username": "postgres",
+			  "password": "postgres"
+			}
+		  ],
+    ```
+    2. It might be necessary to add additional VS Code Extensions to allow for an easier dev experience. This could be database dependent (`MSSQL`, `NoSQL`, etc.)
+4. Modify the existing Tests or add new tests to ensure that the following scenarios are possible:
+    1. The application can successfully ping the DB Container.
+    2. The application can login to the DB Container using the default login data. 
+    3. The application can send a query to the existing DB and receive data from the DB. (Typically listing the databases in the cluster is considered a valid test.)
```

Comment:
are we deploying database clusters? The examples we had were single instance.

Reply:
Apologies, I got confused because of the naming that PostgreSQL uses. I'll make this more clear.